src/invidious/search/filters.cr

```diff
@@ -228,6 +228,25 @@ module Invidious::Search
       return filters
     end
 
+    def to_iv_params : HTTP::Params
+      # Temporary variables
+      raw_params = {} of String => Array(String)
+
+      raw_params["date"] = [@date.to_s.underscore] if !@date.none?
+      raw_params["type"] = [@type.to_s.underscore] if !@type.all?
+      raw_params["sort"] = [@sort.to_s.underscore] if !@sort.relevance?
+
+      if !@duration.none?
+        raw_params["duration"] = [@duration.to_s.underscore]
+      end
+
+      if !@features.none?
+        raw_params["features"] = [Filters.format_features(@features)]
```

Comment:
Should this split on the comma? It's going to look like `["live,4k,3d"]`

Reply:
No, it shoudln't split on the comma, and yup, that's expected :)

Previously, the features filter was passed in the query, as so:
```some search query features:live,4k,3d"```

Imo it's better to keep it that way, even with URL parameters, so the URL can be easily edited by hand, without needing the lenghy `&features=live&features=4k&features=3d....`.